client/src/pages/Bookings/NextBookings.tsx

```diff
@@ -0,0 +1,101 @@
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+// import Avatar from '@material-ui/core/ButtonBase';
+import Typography from '@material-ui/core/Typography';
+import { deepOrange } from '@material-ui/core/colors';
+import EditButton from './EditButton';
+
+interface Props {
+  nextBooking: any;
+}
+
+export default function NextBookings({ nextBooking }: Props): JSX.Element {
+  const pageStyles = makeStyles((theme: Theme) =>
+    createStyles({
+      paper: {
+        padding: theme.spacing(2),
+        margin: 'auto',
+        maxWidth: 500,
+      },
+      image: {
+        width: 128,
+        height: 128,
+      },
+      img: {
+        margin: 'auto',
+        display: 'block',
+        maxWidth: '100%',
+        maxHeight: '100%',
+      },
+      wrapper: {
+        textAlign: 'left',
+        marginBottom: '15px',
+      },
+      sectionHr: {
+        fontWeight: 900,
+        fontSize: '9px',
+        color: 'black',
+        marginBottom: '10px',
+        marginTop: '10px',
+      },
+      nextBookSubHr: {
+        fontWeight: 600,
+        fontSize: '16px',
+        color: 'black',
+      },
+      nextBookNameHr: {
+        fontWeight: 900,
+        fontSize: '13px',
+        color: 'black',
+      },
+      nextBookNameWrapper: {
+        display: 'flex',
+        marginBottom: '15px',
+      },
+      tempImg: {
+        borderRadius: '50%',
+        width: '20px',
+        marginRight: '10px',
+        backgroundColor: deepOrange[500],
+      },
+      icon: {
+        fontSize: '15px',
+        color: 'rgb(209,209,209)',
+      },
+    }),
+  );
+  const pageClasses = pageStyles();
+
+  return (
+    <div className={pageClasses.wrapper}>
+      <Paper className={pageClasses.paper}>
+        <Grid container spacing={2}>
+          <Grid item xs={12} sm container>
+            <Grid item xs container direction="column" spacing={2}>
+              <Grid item xs>
+                <Typography variant="body2" gutterBottom className={pageClasses.sectionHr}>
+                  YOUR NEXT BOOKING
```

Comment:
Please avoid using CAPITACASE as some screen readers interpret them as abbreviations. You can use a CSS class to transform the text instead.

Reply:
okay, done.

Reply:
Please address my comment from above and make the relevant change